package.json

```diff
@@ -44,6 +44,7 @@
     "test:watch": "jest --watch"
   },
   "dependencies": {
+    "@types/axios": "^0.14.0",
```

Comment:
I'm not sure why these got updated, may be deliberate choice, but if not, ensure these changes don't happen. These can be caused due to running `npm upgrade` instead of `npm install` (or yarn equivalent)